controllers/subscriptions.go

```diff
@@ -27,6 +28,9 @@ var bundleInfo []types.Bundle
 func init() {
 	// Reads the yaml file at process startup
 	bundleInfo = GetBundleInfo(config.GetConfig().Options.GetString(config.Keys.BundleInfoYaml))
+	if bundleInfo[0].Error != nil {
+		log.Fatal("Error reading bundles.yml ", zap.Error(bundleInfo[0].Error))
```

Comment:
does `log.Fatal` exit the program?

Reply:
@shdunning Yeah, it outputs something like this:
```
>> starting...
2019/10/14 14:45:22 Error reading bundles.yml {error 25 0  yaml: line 10: could not find expected ':'}
exit status 1
exited: exit status 1
>> restart backoff... 1000ms
```